### PR TITLE
Convert to API, remove mentions of syntax

### PR DESCRIPTION
**Pull request by jridgewell**
At this month's TC39 plenary, several delegates expressed (some strongly) disapproval of the syntax form.

Closes #44 
Closes #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ The examples above would be solved like this:
 ```javascript
 class MyClass {
   print() {
-    console.log(@`
+    console.log(String.dedent`
```
Comment (hemanth) [inline README.md:140]: Merged too soon, I hope @mmkal is ok!

Comment (mmkal) [reply README.md:140]: :’( I’m not ok! But what can you do. @jridgewell are there notes from the plenary?

Comment (jridgewell) [reply README.md:140]: Notes are released after 1 week. Some of the disapproval was on [Matrix](https://github.com/tc39/how-we-work/blob/master/matrix-guide.md), though, with only 2 expression dissent in the meeting. https://matrixlogs.bakkot.com/TC39_Delegates/2022-03-30#L370-L464 is the matrix logs

**Comment by samualtnorman** [comment]
ah this is such a shame

**Pull request merged**